envoy-control-tests/src/main/resources/envoy/echo3_envoy_auth_config.yaml

```diff
@@ -0,0 +1,96 @@
+admin:
```

Comment:
Is this a new config file? Where is it used?

Reply:
Didn't noticed them during merge. Removed